packages/renderer/src/lib/kube/details/KubeIngressArtifact.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+import type { V1IngressSpec } from '@kubernetes/client-node';
+import Cell from './ui/Cell.svelte';
+import Title from './ui/Title.svelte';
+import Link from '../../ui/Link.svelte';
+
+// Props for Ingress artifact and Status
+export let artifact: V1IngressSpec | undefined;
+</script>
+
+{#if artifact}
+  <tr>
+    <Title>Details</Title>
+  </tr>
+
+  {#if artifact.rules}
+    <tr>
+      <Cell>Rules</Cell>
+      <Cell>
+        {#each artifact.rules || [] as rule}
+          <!-- Here we use || [] to ensure it's always an array -->
+          {#if rule.http}
+            {#each rule.http.paths as path}
+              Path: {path.path} • Link:
+              <Link externalRef="{artifact.tls && artifact.tls.length > 0 ? 'https' : 'http'}://{rule.host}{path.path}">
+                {artifact.tls && artifact.tls.length > 0 ? 'https' : 'http'}://{rule.host}{path.path}
+              </Link>
+              • Backend: {path.backend.service?.name}{path.backend.service?.port?.number ? ':' : ''}{path.backend
+                .service?.port?.number}
```

Comment:
Maybe we should check the values here.
I tried to check the summary of the resources i had on my cluster and i saw 
![image](https://github.com/containers/podman-desktop/assets/49404737/e7c67102-c9f0-4b86-8a7f-7faefc388cf4)